config/config.json

```diff
@@ -48,6 +48,10 @@
                 "github.com/keep-network/tbtc-dapp"
             ]
         },
+        "github.com/keep-network/allthekeeps": {
+            "workflow": "allthekeeps-ropsten.yml",
+            "downstream": []
+        },
```

Comment:
Apart from adding config for the `allthekeeps` module, we need to configure the `allthekeeps` module as downstream of some other module (otherwise it will never get executed). I propose to run the `allthekeeps` module as a downstream of the `coverage-pools` module, in parallel with the `keep-core/dashboard` module. Although coverage pools are not supported at the moment by the AllTheeKeeps app (which means we could build the `allthekeeps` even earlier in the flow, after `tbtc/solidity`), they may be supported in the future, so I think it's a good idea to be already prepared for that.

Reply:
done in 9400788